pkg/apis/serving/v1beta1/predictor_torchserve.go

```diff
@@ -66,24 +70,16 @@ func (t *TorchServeSpec) Default(config *InferenceServicesConfig) {
 			t.RuntimeVersion = proto.String(config.Predictors.PyTorch.DefaultImageVersion)
 		}
 	}
-	if t.ModelClassName == "" {
-		t.ModelClassName = DefaultPyTorchModelClassName
-	}
 	setResourceRequirementDefaults(&t.Resources)
 }
 
 // GetContainers transforms the resource into a container spec
 func (t *TorchServeSpec) GetContainer(metadata metav1.ObjectMeta, extensions *ComponentExtensionSpec, config *InferenceServicesConfig) *v1.Container {
 	arguments := []string{
-		fmt.Sprintf("%s=%s", constants.ArgumentModelName, metadata.Name),
-		fmt.Sprintf("%s=%s", constants.ArgumentModelClassName, t.ModelClassName),
-		fmt.Sprintf("%s=%s", constants.ArgumentModelDir, constants.DefaultModelLocalMountPath),
-		fmt.Sprintf("%s=%s", constants.ArgumentHttpPort, constants.InferenceServiceDefaultHttpPort),
-	}
-	if utils.IsGPUEnabled(t.Resources) {
-		arguments = append(arguments, fmt.Sprintf("%s=%s", constants.ArgumentWorkers, "1"))
-	} else if extensions.ContainerConcurrency != nil {
-		arguments = append(arguments, fmt.Sprintf("%s=%s", constants.ArgumentWorkers, strconv.FormatInt(*extensions.ContainerConcurrency, 10)))
+		"torchserve",
```

Comment:
We can switch between v1 protocol and tochserve protocol here like we did for [sklearn](https://github.com/kubeflow/kfserving/blob/master/pkg/apis/serving/v1beta1/predictor_sklearn.go#L66)